app/src/main/res/values/strings.xml

```diff
@@ -186,6 +187,7 @@
     <string name="pref_summary_colored_navigation_bar">Colors the navigation bar in the primary color.</string>
     <string name="pref_summary_colored_app_shortcuts">Colors the app shortcuts in the primary color.</string>
     <string name="pref_summary_audio_ducking">Notifications, navigation etc.</string>
+    <string name="pref_summary_synchronized_lyrics_show">Show synchronized lyrics for song if available</string>
```

Comment:
for lrc file in the music folder, any guide to name the lrc file, or just same as music file name?
and what's the page code for lrc file? utf-8 or?
the translation file should be updated as these commits.
thanks again.